luigi/format.py

```diff
@@ -499,6 +503,72 @@ def pipe_reader(self, input_pipe):
     def pipe_writer(self, output_pipe):
         return OutputPipeProcessWrapper(['bzip2'], output_pipe)
 
+
+class DirectoryFormat(Format):
+    input = 'bytes'
+    output = 'dir'
+
+    # on osx/other env we have different tools, we need the way to overload
+    # should be moved into general configuration or ToolsConfig class.
+    gnu_split = 'split'  # gnu split required (gsplit on OSX)
+    gnu_cat = 'cat'
+    support_bsd_split = True
+
+    def __init__(self, prefix='part-', max_part_size=None, suffix=None, writes_before_flash=None):
+        super(DirectoryFormat, self).__init__()
+        self.max_size = max_part_size
```

Comment:
Can you rename this to `max_part_size` too? It's very confusing when grepping otherwise.